publisher/server_only/README.md

```diff
@@ -15,17 +15,19 @@ docker build . -t uid2-publisher-server
 docker run -it --rm -p 3000:3000 \
     -e UID2_BASE_URL="https://integ.uidapi.com" \
     -e UID2_API_KEY="<your-integ-API-key>" \
+    -e UID2_CLIENT_SECRET="<your-client-secret>" \
     -e SESSION_KEY="my-session-key" \
     uid2-publisher-server
 ```
 
 The following table lists the environment variables that you must specify to start the application.
 
-| Parameter | Data Type | Description |
-| :--- | :--- | :--- |
-| `UID2_BASE_URL` | string | The base URL of the UID2 service. For example:</br>Testing environment: `https://integ.uidapi.com`<br/>Production environment: `https://prod.uidapi.com` |
-| `UID2_API_KEY` | string | Your UID2 authentication key for the UID2 service specified in `UID2_BASE_URL`. |
-| `SESSION_KEY` | string |  The key to the encryption session data stored in application's session cookie. |
+| Parameter            | Data Type | Description                                                                                                                                              |
+|:---------------------| :--- |:---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `UID2_BASE_URL`      | string | The base URL of the UID2 service. For example:</br>Testing environment: `https://integ.uidapi.com`<br/>Production environment: `https://prod.uidapi.com` |
+| `UID2_API_KEY`       | string | Your UID2 authentication key for the UID2 service specified in `UID2_BASE_URL`.|                                                                          |
+| `UID2_CLIENT_SECRET` | string | Your UID2 client secret for the UID2 service specified in `UID2_BASE_URL`.|                                                                           |
+| `SESSION_KEY`        | string | The key to the encryption session data stored in application's session cookie.|                                                                           |
```

Comment:
What value are we getting from the session key? I think it can just be any arbitrary string, and unless you know that you might go hunting trying to find out what value needs to go here.